h2o-dist/index.html

```diff
@@ -493,152 +512,161 @@ <h1><span>H<sub>2</sub>O</span></h1>
 
-                <p>Copy and paste these commands into R one line at a time:</p>
-                  <button class="btn copy_button" id="btnCopy3" data-copied-hint="Copied!" data-clipboard-target='to_copy3'>
+                <p  class="get-started-p">Copy and paste these commands into R one line at a time:</p>
+                  <button class="btn copy_button" id="btnCopy3" data-copied-hint="Copied!" data-clipboard-target="to_copy3">
                     <span class="octicon octicon-clippy"></span>
                   </button>
-                <p class="terminal" id="to_copy3"data-clipboard-target='to_copy3'>
-                    <em># The following two commands remove any previously installed H<sub>2</sub>O packages for R.</em><br/>
-                    if ("package:h2o" %in% search()) { detach("package:h2o", unload=TRUE) }<br/>
-                    if ("h2o" %in% rownames(installed.packages())) { remove.packages("h2o") }<br/>
-                    <br/>
-                    <em># Next, we download packages that H<sub>2</sub>O depends on.</em><br/>
-                    pkgs &lt;- c("RCurl","jsonlite")<br/>
-                    for (pkg in pkgs) {<br/>
-                      if (! (pkg %in% rownames(installed.packages()))) { install.packages(pkg) }<br/>
-                    }<br/>
-                    <br/>
-                    <em># Now we download, install and initialize the H<sub>2</sub>O package for R.</em><br/>
-                    install.packages("h2o", type="source", repos="<script type="text/javascript">var url = location.href; var url2 = url.substring(0, url.lastIndexOf("/")); document.write(url2);</script>/R")<br/>
-                    <br/>
-                    <em># Finally, let's load H2O and start up an H2O cluster</em><br/>
-                    library(h2o)<br/>
-                    h2o.init()<br/>
-            </div>
+                <p class="terminal" id="to_copy3" data-clipboard-target="to_copy3">
+                    <em># The following two commands remove any previously installed H2O packages for R.</em><br>
+                    if ("package:h2o" %in% search()) { detach("package:h2o", unload=TRUE) }<br>
+                    if ("h2o" %in% rownames(installed.packages())) { remove.packages("h2o") }<br>
+                    <br>
+                    <em># Next, we download packages that H2O depends on.</em><br>
+                    pkgs &lt;- c("RCurl","jsonlite")<br>
+                    for (pkg in pkgs) {<br>
+                      if (! (pkg %in% rownames(installed.packages()))) { install.packages(pkg) }<br>
+                    }<br>
+                    <br>
+                    <em># Now we download, install and initialize the H2O package for R.</em><br>
+                    install.packages("h2o", type="source", repos="<script type="text/javascript">var url = location.href; var url2 = url.substring(0, url.lastIndexOf("/")); document.write(url2);</script>http://h2o-release.s3.amazonaws.com/h2o/rel-xia/2/R")<br>
```

Comment:
Don't have rel-xia here.

This has shown up a lot of times now... Probably the right thing is to start over from the original file and only put in the stuff that's actually needed.